zebra-state/src/service/finalized_state/disk_db.rs

```diff
@@ -431,14 +402,19 @@ impl DiskDb {
     /// Returns an iterator over the items in `cf` in `range`.
     ///
     /// Holding this iterator open might delay block commit transactions.
-    pub fn zs_range_iter<C, K, V, R>(&self, cf: &C, range: R) -> impl Iterator<Item = (K, V)> + '_
+    pub fn zs_range_iter<C, K, V, R>(
+        &self,
+        cf: &C,
+        range: R,
+        reverse: bool,
```

Comment:
This new argument needs an update to the method docs.

Reply:
Updated in https://github.com/ZcashFoundation/zebra/pull/7731/commits/a703d1ce996256b02f13d707d51cde17d8d33c64.